lib/ClangImporter/SwiftLookupTable.h

```diff
@@ -133,8 +134,14 @@ class EffectiveClangContext {
     return StringRef(Unresolved.Data, UnresolvedLength);
   }
 };
+
+#if LLVM_PTR_SIZE == 4
+static_assert(sizeof(EffectiveClangContext) <= 4 * sizeof(void *),
+              "should fit in four pointers");
+#else
 static_assert(sizeof(EffectiveClangContext) <= 2 * sizeof(void *),
               "should fit in a couple pointers");
+#endif
```

Comment:
(Originally I think the whole thing fit in one pointer, even on 32-bit platforms. Going from one to two is a bigger jump than two to four.)